rllib/tuned_examples/sac/pendulum-sac.yaml

```diff
@@ -1,13 +1,16 @@
 # Pendulum SAC can attain -150+ reward in 6-7k
 # Configurations are the similar to original softlearning/sac codebase
-pendulum_sac:
+pendulum-sac:
```

Comment:
A lot of these examples are linked from the documentation rst in the algorithms page. Did you make sure none of those links are broken?

Reply:
+1. Checking ...

Reply:
done